src/client/java/net/neoforged/neoforge/client/network/event/RegisterClientPayloadHandlersEvent.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.network.event;
```

Comment:
Is there a reason to add a new event package here over just putting it in the network package?